src/assets/style-utils/colors.scss

```diff
@@ -0,0 +1,51 @@
+// Colors
+// -------------------------
+// Need to define the raw colors somewhere!
+// stylelint-disable color-no-hex, function-max-empty-lines, value-list-max-empty-lines
+$color-palette-data: (
+    white: (base: #fff),
+    black: (base: #000),
+    gray: (base: #444, light: #666, dark: #222)
```

Comment:
If there are say 6 shades here, how do we name them?

Reply:
Up to the author, but these typically are best derived from what's been agreed upon in the Zeplin stylesheet.

Reply:
Suggestion?

Reply:
I use a plugin that calls this api http://chir.ag/projects/ntc/

If we, as devs are responsible for naming the colors, this takes the thought process out and theyre all unique. Maybe this is a requirement from creative though?

Reply:
At HAUS, we're not responsible for naming colors. We work with the designers to name them (or take their lead), establish them in the Zeplin style guide, and then transfer them to our codebase.